### ISSUE TITLE
The Preview Intel Driver link in the README doesn't work anymore; stable driver has WSLg support?

### ISSUE THREAD

**Issue by nelsonjchen** [microsoft/wslg#429]
https://github.com/microsoft/wslg/blob/abed7dfa7e9922dfbd2a57ea6cd48351c21f1f42/README.md#pre-requisites

I think the latest stable drivers have support. I could be wrong though. https://www.intel.com/content/www/us/en/download/19344/intel-graphics-windows-10-windows-11-dch-drivers.html

**Comment by lewisgdljr** [comment]
I can confirm that the latest stable driver works in wslg. So have the last several versions. I'm using it.

**Comment by Mr187k** [comment]
Similar issue here. I installed Intel Graphics drivers for Windows 11 but Ubuntu under WSL can't detect vGPU. Driver version is 30.0.100.9864.
Downloaded from https://www.intel.com/content/www/us/en/download/19344/intel-graphics-windows-10-windows-11-dch-drivers.html

Dxdiag under Windows states WDDM 2.1 strangely

**Comment by nelsonjchen** [comment]
> Similar issue here. I installed Intel Graphics drivers for Windows 11 but Ubuntu under WSL can't detect vGPU. Driver version is 30.0.100.9864.
> Downloaded from https://www.intel.com/content/www/us/en/download/19344/intel-graphics-windows-10-windows-11-dch-drivers.html
> 
> Dxdiag under Windows states WDDM 2.1 strangely

I don't think this is a similar issue. You may want to make a new issue. This issue is about an issue with the current documentation.
